### PR TITLE
Refresh ar-national to 2021

### PR DESCRIPTION
**Pull request by mpoma**

### CODE REVIEW

**Comment by mpoma** [comment]
I refresh Argentina national holidays to 2021

File: src/Cmixin/Holidays/ar-national.php [hunk 1]
```diff
@@ -14,12 +14,19 @@
...
     '06-20'                                              => '06-20',
     'substitutes-06-20'                                  => '= 06-20 if Tuesday then previous Monday and if Thursday then next Friday',
     '07-09'                                              => '07-09',
-    '3rd-monday-of-august'                               => '= third Monday of August',
-    '10-12'                                              => '= 10-12 if Tuesday,Wednesday then previous Monday and if Thursday,Friday,Saturday,Sunday then next Monday',
-    '4th-monday-of-november'                             => '= fourth Monday of November',
+//    '3rd-monday-of-august'                               => '= third Monday of August',
+    'Jose-de-San-Martin'                                 => '= 08-17 if Tuesday,Wednesday then previous Monday and if Thursday,Friday then next Monday',             
+    'day-of-respect-for-cultural-diversity'              => '= 10-12 if Tuesday,Wednesday then previous Monday and if Thursday,Friday then next Monday',
+    'substitutes-day-of-respect-for-cultural-diversity'  => '10-08',    //especial day for 2021
```
Comment (kylekatarnls) [inline src/Cmixin/Holidays/ar-national.php:24]: If it won't be holiday on next years, precise the year:
```suggestion
    'substitutes-day-of-respect-for-cultural-diversity'  => '2021-10-08',    //especial day for 2021
```

File: src/Cmixin/Holidays/ar-national.php [hunk 2]
```diff
@@ -14,12 +14,19 @@
...
+    'day-of-national-sovereignty'                        => '11-20',
+    'substitutes-soberany-day'                           => '= 11-20 if Tuesday,Wednesday then previous Monday and if Thursday,Friday then next Monday',
+    'substitutes-soberany-day'                           => '11-22',    //especial day for 2021
```
Comment (kylekatarnls) [inline src/Cmixin/Holidays/ar-national.php:27]:
```suggestion
    'substitutes-soberany-day'                           => '2021-11-22',    //especial day for 2021
```

**Review by kylekatarnls — CHANGES_REQUESTED** [review]
Thanks 🙏

**Review by kylekatarnls — COMMENTED** [review]
I will check #65 then come back to this.

**Review by kylekatarnls — APPROVED** [review]

**Pull request merged**